### ISSUE TITLE
Mod Request: FerriteCore

### ISSUE THREAD

**Issue by DragonEggBedrockBreaking** [ReperakPro/fabric-optimization-mods#2]
**Mod name:** FerriteCore

**Type of mod:** Memory

**Repository:** https://github.com/malte0811/FerriteCore

**Developer/organization:** malte0811

**Additional notes:**
Technically compatible with Hydrogen, because the code does not conflict, and also a couple (not all) of Hydrogen's mixins are disabled.

**Comment by ReperakPro** [comment]
Denied, due to it using MCP mappings which are non-free.

**Comment by NebelNidas** [comment]
You won't include a mod because its MAPPINGS aren't MIT? Honestly, that's quite stupid, since all of the rest is 100% open-source and the end user won't notice which mappings have been used anyway...

**Comment by DragonEggBedrockBreaking** [comment]
I agree with NebelNidas. This is ridiculous. You can read through the code and make sure it it safe. You can build from source if you dont trust the binaries. You can contribute. For what matters, i.e. what the end user needs, it is open source. So what if the mappings are not open? This mod is amazing, and it is as open as is necessary and as matters for the people who use it. Although this is your repo, this is a public list that is supposed to help the general people.

As more evidence of what is open, here are some "official" definitions of open source:

> pertaining to or denoting software whose source code is available free of charge to the public to use, copy, modify, sublicense, or distribute.

> Open source software is software with source code that anyone can inspect, modify, and enhance.

Finally, this list has no LICENSE.md, therefore it is ARR. Technically, this gist is less open source than FerriteCore, but you are fine with it.

**Comment by Pan00Pernicek** [comment]
Yeah

**Comment by DragonEggBedrockBreaking** [comment]
Eh, I agree with reperak on this one.

> and the end user won't notice which mappings have been used anyway

Most end users don't notice if a mod itself is open source or not, though. This doesn't mean that we should throw closed source mods into this list.

> You won't include a mod because its MAPPINGS aren't MIT?

Well, I know that some smarter people also have other issues with MCP mappings than just being closed, to quote an anonymous source "I wouldn't touch MCP mappings either".

**Comment by NebelNidas** [comment]
You almost sound like the Linux fascists with their retarded "everything that's not 100% open source is evil and must be burnt to the ground" ideology...

**Comment by DragonEggBedrockBreaking** [comment]
> You almost sound like the Linux fascists with their retarded "everything that's not 100% open source is evil and must be burnt to the ground" ideology...

1) your definition of fascism is incorrect
2) MCP is different

**Comment by ReperakPro** [comment]
Okay I'll add this but with an epic poggers 100 warning to let people know that it's MCP :)
